tests/pipeline/test_pipeline.py

```diff
@@ -16,6 +16,7 @@
 PIPELINE_BASE_DIR_PATH = RESOURCE_PATH.parent
 
 
+@pytest.mark.usefixtures("_mock_env")
```

Comment:
Stay consistent with the annotation placement. Either put it on class or method.

Reply:
I wasn't sure whether future tests in the `TestExample` class would benefit from the fixture, so I only enabled it for the 1 test inside.

Here there were too many tests, so I opted for decorating the whole class.

Reply:
I'll move the one in `test_example`